RecoLocalCalo/HGCalRecAlgos/src/HGCalImagingAlgo.cc

```diff
@@ -491,9 +499,9 @@ void HGCalImagingAlgo::shareEnergy(const std::vector<KDNode>& incluster,
     outclusters[i].resize(incluster.size(),0.0);
     for( unsigned j = 0; j < incluster.size(); ++j ) {
       if( j == seeds[i] ) {
-	outclusters[i][j] = 1.0;
-	centroids[i] = math::XYZPoint(incluster[j].data.x,incluster[j].data.y,incluster[j].data.z);
-	energies[i]  = incluster[j].data.weight;
+    outclusters[i][j] = 1.0;
+    centroids[i] = math::XYZPoint(incluster[j].data.x,incluster[j].data.y,incluster[j].data.z);
+    energies[i]  = incluster[j].data.weight;
       }
```

Comment:
the indentation is off here as well
https://github.com/felicepantaleo/cmssw/blob/4a4de46f104e5240ed02977bf2526175a894ed43/RecoLocalCalo/HGCalRecAlgos/src/HGCalImagingAlgo.cc#L501-L505